tests/test_cmdline.py

```diff
@@ -48,8 +48,6 @@ def test_cmdline(design,num_placements,PDN_mode,select_in_ILP):
     args = [str(design_dir), '-f', str(design_dir / f"{design}.sp"), '-s', design, '-p', str(pdk_dir), '-flat',  str(0), '-v', 'INFO', '-l', 'INFO', '-n', str(num_placements)]
     if PDN_mode:
         args.append( '--PDN_mode')
-    if select_in_ILP:
-        args.append( '--select_in_ILP')
```

Comment:
Why this removal?

Reply:
My mistake. All these removal for debugging. I'm reverting these.